experiment/revert-bump.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+bold_blue() {
+  echo -e "\x1B[1;34m${*}\x1B[0m"
+}
+
+bold_yellow() {
+  echo -e "\x1B[1;33m${*}\x1B[0m"
+}
+
+main() {
+  bold_blue "Checking out latest upstream ..."
+  git fetch https://github.com/kubernetes/test-infra master
+  git checkout FETCH_HEAD
+
+  bold_blue "Finding most recent bump commit ..."
+  local most_recent_bump
+  local most_recent_bump_commmit
+  most_recent_bump="$(git log --no-merges --extended-regexp \
```

Comment:
Why `--no-merges`? When I revert a PR, I revert the merge. Seems cleaner. /shrug

Reply:
I'm matching on the known message format we control in the actual bump commit, not searching these means we grep against less commits.

Reverting this versus the merge makes no functional difference ...?

Reply:
I tend to revert commits rather than merge commits. So this marches my behavior

Reply:
I tend to revert the merge.

I don't think it matters at all.